tests/Service/FilterFactoryTest.php

```diff
@@ -55,8 +59,10 @@ public function provideFilters(): array
             'gte' => ['gte', FilterWithOperator::class],
             'lt' => ['lt', FilterWithOperator::class],
             'lte' => ['lte', FilterWithOperator::class],
-            'in' => ['in', FilterIn::class, self::COLUMN . '_in', 'value', ['value']],
+            'in' => ['in', FilterIn::class, null, 'value', ['value']],
```

Comment:
I don't like the nullable params and placing the logic to the test body: it is prone to error and the provider is hard to read and not explicit. I would preffer having eg. "column_in" (even without using the constant) as the expectedTitle value.

Reply:
I know its out of scope of this PR, but would be nice to unify it amongst all the datasets :).

Reply:
Well.. could you send PR on that? Because I'm really confused from what you want or not.. you wanted me to provide only different values for each test case here https://github.com/lmc-eu/api-filter/pull/31#discussion_r229259344 but now you want to pass all values always? :confused: or maybe I dont correctly understand what you want..